src/rhsm_debug/debug_commands.py

```diff
@@ -205,7 +204,7 @@ def _do_command(self):
                 print(_("Wrote: %s") % dest_dir_name)
 
         except Exception as e:
-            managercli.handle_exception(_("Unable to create zip file of system information: %s") % e, e)
+            handle_exception(_("Unable to create zip file of system information: %s") % e, e)
```

Comment:
Why use %s when building the string?  .format() or a f-string would be more readable more pythonic.